Interfaces/MainMenu.xib

```diff
@@ -31,7 +31,7 @@
                             <menuItem isSeparatorItem="YES" id="Dxk-DH-qzi">
                                 <modifierMask key="keyEquivalentModifierMask" command="YES"/>
                             </menuItem>
-                            <menuItem title="Check For Updates…" identifier="Check For Updates…" id="880">
+                            <menuItem title="Check for Updates…" identifier="Check for Updates…" id="880">
```

Comment:
Don't change the identifier. Existing scripts rely on this being stable to continue working.